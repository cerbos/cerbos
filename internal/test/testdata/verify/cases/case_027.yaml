description: Invalid aux data fixtures
