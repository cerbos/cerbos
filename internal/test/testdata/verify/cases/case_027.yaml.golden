{
  "suites": [
    {
      "file": "suite_test.yaml",
      "name": "TestSuite",
      "summary": {
        "overallResult": "RESULT_ERRORED"
      },
      "error": "failed to load test fixtures from testdata: file is not valid: { /auxData/myJWT: [additionalProperties 'jort' not allowed] }",
      "description": "Tests for verifying something"
    }
  ],
  "summary": {
    "overallResult": "RESULT_ERRORED"
  }
}
